aio/src/styles/1-layouts/top-menu/_top-menu.scss

```
@@ -132,7 +132,10 @@ mat-toolbar.app-toolbar {
 
           .nav-link-inner {
             border-radius: 4px;
+            font-size: 1.3rem;
             padding: 8px 16px;
+            font-size: clamp(10px, 1.6rem, 20px);
```

Nit: Could we put the overriding styles immediately after the ones they override (to increase the chances of them staying in sync 😅):

```suggestion
            font-size: clamp(10px, 1.6rem, 20px);
            padding: 8px 16px;
```

---

ah yeah good point :slightly_smiling_face: